impeller/aiks/canvas_benchmarks.cc

```diff
@@ -0,0 +1,60 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/benchmarking/benchmarking.h"
+
+#include "impeller/aiks/canvas.h"
+
+namespace impeller {
+
+namespace {
+size_t DrawRect() {
+  Canvas canvas;
+  for (auto i = 0; i < 500; i++) {
+    canvas.DrawRect(Rect::MakeLTRB(0, 0, 100, 100),
+                    {.color = Color::DarkKhaki()});
+  }
+  return 500;
+}
+
+size_t DrawCircle() {
+  Canvas canvas;
+  for (auto i = 0; i < 500; i++) {
+    canvas.DrawCircle({100, 100}, 5, {.color = Color::DarkKhaki()});
+  }
+  return 500;
+}
+
+size_t DrawLine() {
+  Canvas canvas;
+  for (auto i = 0; i < 500; i++) {
+    canvas.DrawLine({0, 0}, {100, 100}, {.color = Color::DarkKhaki()});
+  }
+  return 500;
+}
```

Comment:
These might get optimized to just returning 500.  I think taking in a pointer to a Canvas that will do the DrawLine operations is a safer alternative.

Reply:
Done